src/ipaperftest/core/constants.py

```diff
@@ -545,3 +562,45 @@ def getLevelName(level):
       line: selinux_provider = none
       insertafter: id_provider = ipa
 """
+
+ANSIBLE_GROUPSIZETEST_SERVER_CONFIG_PLAYBOOK = """
+---
+- name: Configure server before execution
+  hosts: ipaserver
+  become: yes
+  tasks:
+    - copy:
+        dest: /etc/ipa/server.conf
+        content: |
+          [global]
+          debug=True
+    - command:
+        cmd: "systemctl restart httpd"
```

Comment:
You could use the service module instead: https://docs.ansible.com/ansible/2.9/modules/service_module.html